tests/test_integrated_channels/test_cornerstone/test_views.py

```diff
@@ -96,7 +95,7 @@ def test_course_list(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(response.data), 3)
-        keys = set([key for item in response.data for key in item.keys()])
+        keys = {key for item in response.data for key in item.keys()}
```

Comment:
???

Reply:
It is done to resolve pylint error R1718: (consider-using-set-comprehension)

https://python-reference.readthedocs.io/en/latest/docs/comprehensions/set_comprehension.html